test_common/harness/kernelHelpers.c

```diff
@@ -75,10 +75,10 @@ std::vector<char> get_file_content(const std::string &fileName)
 
 std::string get_kernel_name(const std::string &source)
 {
-    cl_uint crc = 0x04c11db7;
+    cl_uint crc = ~0U;
```

Comment:
The code could be simplified slightly by moving this into `crc32`. Otherwise, looks good to me.

Reply:
Note that in `crc32()`, we do `crc = crc ^ ~0U;`. Propagation of the value of `~0U` here into `crc32()` would give `crc = ~0U ^ ~0U`, i.e. `crc = 0`, which is incorrect, so this line should be removed.

Reply:
Fixed